### ISSUE TITLE
Add Plausible Analytics to website 

### ISSUE THREAD

**Issue by andrewtavis** [activist-org/activist#114]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/activist-org/activist/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

activist is going to be adding in simple analytics so we have a better understanding of the site traffic. To achieve this we'll be using [Plausible Analytics](https://plausible.io/). We'll for now use the cloud version and potentially/hopefully switch to self hosted in the future when we have more capacity.

The following script needs to be put onto the website for this to work:

```
<script defer data-domain="activist.org" src="https://plausible.io/js/script.js"></script>
```

A bit more research is needed on where exactly this needs to go. My assumption is that putting it into the <head> of the [layout files](https://github.com/activist-org/activist/tree/main/frontend/layouts) will then propagate it to all the pages and give us the information we'd need.

### Contribution

I'll work on this in the coming week :)

**Comment by andrewtavis** [comment]
@therealhoppi, FYI made an issue for this :) As stated in the issue I need to check to see if adding it to the layouts will be enough or if it needs to be explicitly added to the pages.

**Comment by andrewtavis** [comment]
Note that there's also a [Nuxt Module for Plausible](https://github.com/nuxt-modules/plausible).

**Comment by andrewtavis** [comment]
@therealhoppi, the changes in 07c97e2 should supposedly get all this set up :) Let me know if anything starts coming through.

**Comment by andrewtavis** [comment]
Just confirmed that this was closed by 07c97e2 :)
